assets/app/views/_project-page.html

```diff
@@ -16,10 +16,8 @@
     <div class="container-fluid">
       <div class="sidebar-header sidebar-header-bleed-left sidebar-header-bleed-right">
         <div class="actions pull-right">
-          <!-- If we have any header actions for the sidebar in the future, they would go here
-              and be links like below
-              <a href="#">Example</a>
-          -->
+	  <a href="#" ng-click="overviewToggle()" ng-if="overviewMode == 'tiles'">Topology</a>
```

Comment:
> Looks like only of these is visible at a time, might be nice if the active one was still visible but looked selected. Also did you consider icons instead of text links?

We are actually having a similar discussion on icon for topology in ManageIQ and have several options for it.
@serenamarie125 can you please update on the status of the icon?